### PR TITLE
Grammar fixes and updates.

### PR DESCRIPTION
**Pull request by rowild**
Mainly fix links and add new infos about updated APIs or packages.
The section about *Authorize-net*specifically needs you review – thank you!

### CODE REVIEW

File: src/manual/services.md [hunk 1]
```diff
@@ -80,12 +80,14 @@ The **PostPay** provider is useful for all payments where the customers pay afte
...
 # Supported by ai-payments
 
-Service providers for many payment gateways are available by the Aimeos [ai-payments extension](https://github.com/aimeoscom/ai-payments). This includes:
-
+Service providers for many payment gateways are available by the *Aimeos* [ai-payments extension](https://github.com/aimeoscom/ai-payments). It includes:
 
 ## Authorize.net AIM
 
-The [Authorize.net](https://www.authorize.net/) gateway for the [AIM method](https://www.authorize.net/support/AIM_guide.pdf) (collect payment details locally and send them to the payment gateway) is available via the **AuthorizeAIM** payment service provider since 2015.07.
+::: hint "Update November 2020:"
```
Comment (aimeos) [inline src/manual/services.md:87]: What does `:::` do?

**Comment by aimeos** [comment]
It would be also a good idea to add `ai-payments` to the `composer req ...` lines

**Comment by aimeos** [comment]
My initial suggestion was to use lines like this one:

**Comment by rowild** [comment]
Fixed.

**Comment by aimeos** [comment]
What are about the `!!! hint "Update November 2020:"` lines?

**Comment by rowild** [comment]
What about them? 
They are meant to be simple call-outs that inform about a docu updates, in this case due to a change in external sources.
How would you do that?

**Comment by aimeos** [comment]
Like written:

> I would rather use:
>
> !!! warn
>
> with no date because it's deprecated and it doesn't matter when the notice have been added

**Comment by rowild** [comment]
Where did you write that?

**Comment by rowild** [comment]
Sorry - "service details" will come as a separate PR.

**Comment by aimeos** [comment]
Great! Thanks for your updates

**Pull request merged**